plugins/plugin-ssh-key/che-plugin-ssh-key-server/src/main/java/org/eclipse/che/plugin/ssh/key/HttpSshServiceClient.java

```diff
@@ -84,8 +81,7 @@ public void createPair(SshPairDto sshPair) throws ServerException {
     @Override
     public SshPairDto getPair(String service, String name) throws ServerException, NotFoundException {
         try {
-            final String url = UriBuilder.fromUri(sshUrl)
-                                         .path(SshService.class, "getPair")
+            final String url = UriBuilder.fromUri(sshUrl + "/getPair")
```

Comment:
Think about adding test dependency on api-ssh module and add test with mock for SshService